packages/aws-cdk/test/api/hotswap-test-setup.ts

```diff
@@ -0,0 +1,51 @@
+import * as cxapi from '@aws-cdk/cx-api';
+import { CloudFormation } from 'aws-sdk';
+import { testStack, TestStackArtifact } from '../util';
+import { MockSdkProvider } from '../util/mock-sdk';
+import { FakeCloudformationStack } from './fake-cloudformation-stack';
+
+const STACK_NAME = 'withouterrors';
+export const STACK_ID = 'stackId';
+
+export let mockSdkProvider: MockSdkProvider;
+export let currentCfnStack: FakeCloudformationStack;
```

Comment:
Gotta say, exporting mutable variables makes me a little queasy.

Any chance we can hide these behind a nicer abstraction?